Fix library management

mastercoms opened the pull request:
Revert back to the old Java 8 classpath behavior because of Java 9 release delays (fixes libraries not being added to classpath properly in some cases)

Add new libraries for increased compatibility (we didn't add the new logging library Mojang uses)

This fixes #500

turt2live commented:
@Tonodus This pull request is no longer mergeable. Please resolve the issue as soon as possible, thanks!

dequis commented:
The hold for this one isn't really needed, is it? Because I suspect it might be a while until the spreadsheet covers absolutely everything

turt2live commented:
I actually completely forgot about that spreadsheet. I'll work on a better way to generate a list of values though because manual methods are... hard.

mastercoms committed:
Fix library management

Revert back to the old Java 8 classpath behavior because of Java 9 delays
Add new libraries for increased compatiblity
